### DISCUSSION TITLE
Database stuck in restoration process for 3 days

### DISCUSSION THREAD

**Discussion by Wildfireblogs** [supabase/supabase#13526] [Questions] [answered]
My database got automatically paused due to inactivity and now I am trying to restore but it seems stuck.

I tried restoring 3 days ago and it was showing the restoration spinners so I left after 15 mins thinking that the restoration will eventually finish on its own but its been 3 days.

here's what I tried:
1. Logging out and in and clearing browser cache/cookies
2. Refreshing the page
3. Pausing project fails
4. Restarting the project

**Comment by GaryAustin1** [comment, marked as answer]
You asked this on Discord also.  You really need to contact support at this point.

**Reply by Wildfireblogs** [reply]
Already contacted. Thanks.
